uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader.php

```diff
@@ -85,9 +86,72 @@ public function install() {
     $this->deleteUnneededCustomGroups();
     $this->createDefaultRelationshipTypes();
     $this->makeAllCurrenciesAvailable();
+    $this->enableComponentsAndFilterDefaultComponents();
     $this->runAllUpgraders();
   }
 
+  /**
+   * Enables required components and remove unwanted core components
+   * from the default components list.
+   */
+  public function enableComponentsAndFilterDefaultComponents() {
+    $this->setComponentsStatus();
+  }
+
+  /**
+   * Enables required components.
+   */
+  public function enableComponents() {
+    $this->setComponentsStatus(TRUE, FALSE);
+  }
+
+  /**
+   * Disables required components.
+   */
+  public function disableComponents() {
+    $this->setComponentsStatus(FALSE, FALSE);
```

Comment:
Why did the signature of this method change? Compared to the previous version we had in `hrcase`, this new one is worse in terms of readability. Previously, we would have something like `setComponentStatus(['CiviCase' => true, 'CiviTask' => false])` and I could easily understand that this would enable CiviCase and disable CiviTask. Now, I cannot understand what is going on unless I actually read the method implementation, because these booleans params simply don't convey any information